### PR TITLE
issue #425, resolve cpp check

### PR DESCRIPTION
**Pull request by avan989**
**Describe the contribution**
resolve cpp check

**Testing performed**
Steps taken to test the contribution:
1. cppcheck --force --inline-suppr --std=c99 --language=c --error-exitcode=1 --enable=warning,performance,portability,style --suppress=variableScope --inconclusive fsw/cfe-core/src 2>cppcheck_flight_cfe.txt

**System(s) tested on:**
 - Hardware
 - Ubuntu 18.04
 - CFE 6.6

**Additional context**
Add any other context about the contribution here.

**Contributor Info**
Anh Van, NASA Goddard

### CODE REVIEW

File: fsw/cfe-core/src/sb/cfe_sb_util.c [hunk 1]
```diff
@@ -109,6 +109,7 @@ void CFE_SB_InitMsg(void           *MsgPtr,
 */
 uint16 CFE_SB_MsgHdrSize(const CFE_SB_Msg_t *MsgPtr)
 {
+    /* cppcheck-suppress unusedVariable */
     uint16 size;
 
 #ifdef MESSAGE_FORMAT_IS_CCSDS
```
Comment (skliper) [inline fsw/cfe-core/src/sb/cfe_sb_util.c]: Can suppression be removed if `uint16 size;` is inside `#ifdef`?

File: fsw/cfe-core/src/sb/cfe_sb_priv.c [hunk 2]
```diff
@@ -705,6 +705,7 @@ char *CFE_SB_GetAppTskName(uint32 TaskId,char *FullName){
 */
 uint8 CFE_SB_GetPktType(CFE_SB_MsgId_t MsgId)
 {
+    /* cppcheck-suppress unreadVariable */
     CFE_SB_MsgId_Atom_t Val = MsgId;
 
 #ifdef MESSAGE_FORMAT_IS_CCSDS
```
Comment (skliper) [inline fsw/cfe-core/src/sb/cfe_sb_priv.c]: Can suppression be removed if line is moved within ifdef?

File: fsw/cfe-core/src/sb/cfe_sb_init.c [hunk 3]
```diff
@@ -217,6 +217,7 @@ void CFE_SB_InitPipeTbl(void){
 */
 void CFE_SB_InitMsgMap(void){
 
+    /* cppcheck-suppress unusedVariable */
     CFE_SB_MsgKey_Atom_t   KeyVal;
 
 #ifdef MESSAGE_FORMAT_IS_CCSDS
```
Comment (skliper) [inline fsw/cfe-core/src/sb/cfe_sb_init.c]: Can suppression be removed if line is within ifdef?

**Comment by avan989** [comment]
updated.

**Comment by skliper** [comment]
CCB 20190108 - Reviewed and approved for integration candidate

**Pull request merged**